ocpmodels/common/relaxation/ase_utils.py

```diff
@@ -169,6 +169,7 @@ def __init__(
             local_rank=config.get("local_rank", 0),
             is_debug=config.get("is_debug", True),
             cpu=cpu,
+            amp=True if config.get("amp", False) else False,
```

Comment:
just `config.get("amp", False)`?

Reply:
config["amp"] is a dictionary not a boolean

Reply:
Let me know if I'm missing something, but `config["amp"]` should be a boolean. Specifically:

```python
ckpt = torch.load(checkpoint_path)
assert type(ckpt["config"]["amp"]) == bool
assert type(ckpt["amp"]) == dict
```

Reply:
Ah sorry I'm mixing the two, yes you're correct! Will update.